registries/adapters/apiv2_adapter.go

```diff
@@ -0,0 +1,220 @@
+//
+// Copyright (c) 2018 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package adapters
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/automationbroker/bundle-lib/bundle"
+	"github.com/automationbroker/bundle-lib/registries/adapters/oauth"
+	log "github.com/sirupsen/logrus"
+)
+
+const apiV2ManifestURL = "%v/v2/%v/manifests/%v"
+const apiV2CatalogURL = "%v/v2/_catalog"
```

Comment:
Do we really need to call it the `apiV2CatalogURL`? isn't it just the catalog url? It is part of the APIV2Adapter presumably we would  never have an apiV3CatalogURL be part of the APIV2Adapter.

I'd keep these just as catalogURL and manifestURL.